### PR TITLE
Arts 267 changedb to mssql

### PR DESCRIPTION
**Pull request by shajidissa**

### CODE REVIEW

File: trial-config-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -5,9 +5,7 @@ spring.jpa.hibernate.ddl-auto=none
...
 
-
 # Log properties in application.properties
-#logging.level.org.springframework.security=DEBUG
-
-# TODO ARTS-264 put these values into Azure Key Vault
-
+logging.level.org.springframework.security=DEBUG
+logging.level.org.hibernate.SQL=DEBUG
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
```
Comment (mdp0) [inline trial-config-service/src/main/resources/application.properties:11]: Might want to tone these down a bit now you've got it working

File: trial-config-service/src/main/resources/db/changelog/changelog-schema.xml [hunk 2]
```diff
@@ -115,13 +115,13 @@
     </changeSet>
     <changeSet author="katesan (generated)" id="1608062861281-9">
         <createTable tableName="revinfo">
-            <column name="rev" type="INTEGER">
-                <constraints nullable="false" primaryKey="true" primaryKeyName="revinfo_pkey"/>
-            </column>
+<!--            <column name="rev" type="INTEGER">-->
+<!--                <constraints nullable="false" primaryKey="true" primaryKeyName="revinfo_pkey"/>-->
+<!--            </column>-->
             <column name="revtstmp" type="BIGINT"/>
```
Comment (mdp0) [inline trial-config-service/src/main/resources/db/changelog/changelog-schema.xml:121]: Dupe comment

**Review by mdp0 — COMMENTED** [review]
Couple of tweaks - debug levels still high

File: trial-config-service/src/main/resources/db/changelog/changelog-schema.xml (hunk 2 above)
Comment (shajidissa) [reply trial-config-service/src/main/resources/db/changelog/changelog-schema.xml:121]: Done

File: trial-config-service/src/main/resources/application.properties (hunk 1 above)
Comment (shajidissa) [reply trial-config-service/src/main/resources/application.properties]: Done

**Review by darrengsmithson — APPROVED** [review]
All good - Ive tested the new server (following the README) against current functionality and it works as expected.

**Comment by darrengsmithson** [comment]
Merging into feature

**Pull request merged**